### PR TITLE
Challenge 0 Simple NFT - AddressInput Fix

### PR DESCRIPTION
**Pull request by Chris-Kag**
Awaiting the eth-hooks JS update

### CODE REVIEW

File: packages/react-app/src/components/AddressInput.jsx [hunk 1]
```diff
@@ -1,29 +1,25 @@
 import { CameraOutlined, QrcodeOutlined } from "@ant-design/icons";
-import { Badge, Input } from "antd";
-import { useLookupAddress } from "eth-hooks/dapps/ens";
```
Comment (adnfx2) [inline packages/react-app/src/components/AddressInput.jsx:3]: I was just starting the ethereumSpeedRun and found out that this change it's braking my fresh build D:

File: packages/react-app/src/components/AddressInput.jsx [hunk 2]
```diff
@@ -1,29 +1,25 @@
...
+import { Badge, Input, message, Spin } from "antd";
+import { useLookupAddress } from "eth-hooks";
```
Comment (adnfx2) [inline packages/react-app/src/components/AddressInput.jsx:3]: I was just starting the ethereumSpeedRun and found out that this change was braking my fresh build D:

Comment (MattAHarrington) [reply packages/react-app/src/components/AddressInput.jsx:3]: sameee

Comment (Chris-Kag) [reply packages/react-app/src/components/AddressInput.jsx:3]: My bad let me fix that

Comment (Chris-Kag) [reply packages/react-app/src/components/AddressInput.jsx]: Just added a temporary fix but as soon as eth-hooks will be updated the addressInput.jsx will change

**Pull request closed without merging**